zboxcore/zboxutil/http.go

```diff
@@ -542,9 +543,18 @@ func MakeSCRestAPICall(scAddress string, relativePath string, params map[string]
 				responses[response.StatusCode]++
 				if responses[response.StatusCode] > maxCount {
 					maxCount = responses[response.StatusCode]
+				}
+
+				if responses[response.StatusCode] > maxCount {
```

Comment:
why is it duplicated?

Reply:
Just a merge issue. Removed the duplicate